From e4c0ae0d133f48f207321539bb34438c8a8af487 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Sun, 4 Sep 2016 02:49:05 -0700
Subject: [PATCH 033/249] cm: Multiple updates

 * Add CodeAurora telephony repo
 * Add ipsec-utils fork
 * Clean redundant remote tags
 * Remove caf.xml

Change-Id: Ia56587c4853fadf5b776af529aad0db69725d8c5
---
 caf.xml              |  71 ------------------
 default.xml          |   2 +-
 snippets/cm-14.0.xml | 167 ++++++++++++++++++++++---------------------
 3 files changed, 85 insertions(+), 155 deletions(-)
 delete mode 100644 caf.xml

diff --git a/caf.xml b/caf.xml
deleted file mode 100644
index 2c83c60..0000000
--- a/caf.xml
+++ /dev/null
@@ -1,71 +0,0 @@
-<manifest>
-
-  <remove-project name="CyanogenMod/android_build" />
-  <remove-project name="CyanogenMod/android_bionic" />
-  <remove-project name="CyanogenMod/android_external_skia" />
-  <remove-project name="CyanogenMod/android_external_wpa_supplicant_8" />
-  <remove-project name="CyanogenMod/android_hardware_libhardware" />
-  <remove-project name="CyanogenMod/android_hardware_libhardware_legacy" />
-  <remove-project name="CyanogenMod/android_frameworks_av" />
-  <remove-project name="CyanogenMod/android_frameworks_base" />
-  <remove-project name="CyanogenMod/android_frameworks_native" />
-  <remove-project name="CyanogenMod/android_frameworks_opt_net_ims" />
-  <remove-project name="CyanogenMod/android_frameworks_opt_net_wifi" />
-  <remove-project name="CyanogenMod/android_frameworks_opt_telephony" />
-  <remove-project name="CyanogenMod/android_packages_apps_Bluetooth" />
-  <remove-project name="CyanogenMod/android_packages_apps_CarrierConfig" />
-  <remove-project name="CyanogenMod/android_packages_apps_CellBroadcastReceiver" />
-  <remove-project name="CyanogenMod/android_packages_apps_Contacts" />
-  <remove-project name="CyanogenMod/android_packages_apps_ContactsCommon" />
-  <remove-project name="CyanogenMod/android_packages_apps_DeskClock" />
-  <remove-project name="CyanogenMod/android_packages_apps_Dialer" />
-  <remove-project name="CyanogenMod/android_packages_apps_Nfc" />
-  <remove-project name="CyanogenMod/android_packages_apps_PhoneCommon" />
-  <remove-project name="CyanogenMod/android_packages_apps_Settings" />
-  <remove-project name="CyanogenMod/android_packages_apps_Stk" />
-  <remove-project name="CyanogenMod/android_packages_providers_ContactsProvider" />
-  <remove-project name="CyanogenMod/android_packages_providers_MediaProvider" />
-  <remove-project name="CyanogenMod/android_packages_providers_TelephonyProvider" />
-  <remove-project name="CyanogenMod/android_packages_services_Telecomm" />
-  <remove-project name="CyanogenMod/android_packages_services_Telephony" />
-  <remove-project name="CyanogenMod/android_system_bt" />
-  <remove-project name="CyanogenMod/android_system_core" />
-  <remove-project name="CyanogenMod/android_system_media" />
-  <remove-project name="CyanogenMod/android_system_vold" />
-
-  <project path="build" name="CyanogenMod/android_build" groups="pdk,tradefed" revision="staging/cm-14.0-caf">
-    <copyfile src="core/root.mk" dest="Makefile" />
-  </project>
-  <project path="bionic" name="CyanogenMod/android_bionic" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="external/skia" name="CyanogenMod/android_external_skia" groups="pdk,qcom_msm8x26" revision="staging/cm-14.0-caf" />
-  <project path="external/wpa_supplicant_8" name="CyanogenMod/android_external_wpa_supplicant_8" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="frameworks/av" name="CyanogenMod/android_frameworks_av" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="frameworks/base" name="CyanogenMod/android_frameworks_base" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="frameworks/native" name="CyanogenMod/android_frameworks_native" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="frameworks/opt/net/ims" name="CyanogenMod/android_frameworks_opt_net_ims" groups="frameworks_ims,pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="frameworks/opt/net/wifi" name="CyanogenMod/android_frameworks_opt_net_wifi" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="frameworks/opt/telephony" name="CyanogenMod/android_frameworks_opt_telephony" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="hardware/libhardware" name="CyanogenMod/android_hardware_libhardware" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="hardware/libhardware_legacy" name="CyanogenMod/android_hardware_libhardware_legacy" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/Bluetooth" name="CyanogenMod/android_packages_apps_Bluetooth" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/CarrierConfig" name="CyanogenMod/android_packages_apps_CarrierConfig" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/CellBroadcastReceiver" name="CyanogenMod/android_packages_apps_CellBroadcastReceiver" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/Contacts" name="CyanogenMod/android_packages_apps_Contacts" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/ContactsCommon" name="CyanogenMod/android_packages_apps_ContactsCommon" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/DeskClock" name="CyanogenMod/android_packages_apps_DeskClock" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/Dialer" name="CyanogenMod/android_packages_apps_Dialer" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/Nfc" name="CyanogenMod/android_packages_apps_Nfc" groups="apps_nfc,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/PhoneCommon" name="CyanogenMod/android_packages_apps_PhoneCommon" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/Settings" name="CyanogenMod/android_packages_apps_Settings" groups="pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/apps/Stk" name="CyanogenMod/android_packages_apps_Stk" groups="apps_stk,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/providers/ContactsProvider" name="CyanogenMod/android_packages_providers_ContactsProvider" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/providers/MediaProvider" name="CyanogenMod/android_packages_providers_MediaProvider" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/providers/TelephonyProvider" name="CyanogenMod/android_packages_providers_TelephonyProvider" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/services/Telecomm" name="CyanogenMod/android_packages_services_Telecomm" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="packages/services/Telephony" name="CyanogenMod/android_packages_services_Telephony" groups="pdk-cw-fs,pdk-fs" revision="staging/cm-14.0-caf" />
-  <project path="system/bt" name="CyanogenMod/android_system_bt" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="system/core" name="CyanogenMod/android_system_core" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="system/media" name="CyanogenMod/android_system_media" groups="pdk" revision="staging/cm-14.0-caf" />
-  <project path="system/vold" name="CyanogenMod/android_system_vold" groups="pdk" revision="staging/cm-14.0-caf" />
-
-</manifest>
diff --git a/default.xml b/default.xml
index 7f2163c..a7ed686 100644
--- a/default.xml
+++ b/default.xml
@@ -135,7 +135,7 @@
   <project path="external/hyphenation-patterns" name="platform/external/hyphenation-patterns" groups="pdk-fs" remote="aosp" />
   <project path="external/icu" name="CyanogenMod/android_external_icu" groups="pdk" />
   <project path="external/iproute2" name="CyanogenMod/android_external_iproute2" groups="pdk" />
-  <project path="external/ipsec-tools" name="platform/external/ipsec-tools" groups="pdk-cw-fs,pdk-fs" remote="aosp" />
+  <project path="external/ipsec-tools" name="CyanogenMod/android_external_ipsec-tools" groups="pdk-cw-fs,pdk-fs" />
   <project path="external/iptables" name="CyanogenMod/android_external_iptables" groups="pdk-cw-fs,pdk-fs" />
   <project path="external/iputils" name="platform/external/iputils" groups="pdk-cw-fs,pdk-fs" remote="aosp" />
   <project path="external/iw" name="platform/external/iw" remote="aosp" />
diff --git a/snippets/cm-14.0.xml b/snippets/cm-14.0.xml
index 53ad00a..79db9b4 100644
--- a/snippets/cm-14.0.xml
+++ b/snippets/cm-14.0.xml
@@ -5,89 +5,90 @@
   <include name="snippets/hal_cm_all.xml" />
 
   <!-- CyanogenMod additions -->
-  <project path="android" name="CyanogenMod/android" remote="github" />
-  <project path="external/ahbottomnavigation" name="CyanogenMod/android_external_ahbottomnavigation" remote="github" />
-  <project path="external/android-visualizer" name="CyanogenMod/android_external_android-visualizer" remote="github" />
-  <project path="external/bash" name="CyanogenMod/android_external_bash" remote="github" />
-  <project path="external/cmsdk-api-coverage" name="CyanogenMod/android_external_cmsdk-api-coverage" remote="github" />
-  <project path="external/exfat" name="CyanogenMod/android_external_exfat" remote="github" />
-  <project path="external/ffmpeg" name="CyanogenMod/android_external_ffmpeg" remote="github" />
-  <project path="external/fuse" name="CyanogenMod/android_external_fuse" remote="github" />
-  <project path="external/gello-build" name="CyanogenMod/android_external_gello_build" remote="github" />
-  <project path="external/google" name="CyanogenMod/android_external_google" remote="github" />
-  <project path="external/htop" name="CyanogenMod/android_external_htop" remote="github" />
-  <project path="external/koush/AndroidAsync" name="CyanogenMod/AndroidAsync" remote="github" />
-  <project path="external/koush/ion" name="CyanogenMod/ion" remote="github" />
-  <project path="external/libncurses" name="CyanogenMod/android_external_libncurses" remote="github" />
-  <project path="external/libphonenumbergoogle" name="CyanogenMod/android_external_libphonenumbergoogle" remote="github" />
-  <project path="external/libtar" name="CyanogenMod/android_external_libtar" remote="github" />
-  <project path="external/libtruezip" name="CyanogenMod/android_external_libtruezip" remote="github" />
-  <project path="external/lzo" name="CyanogenMod/android_external_lzo" remote="github" />
-  <project path="external/nano" name="CyanogenMod/android_external_nano" remote="github" />
-  <project path="external/ntfs-3g" name="CyanogenMod/android_external_ntfs-3g" remote="github" />
-  <project path="external/openssh" name="CyanogenMod/android_external_openssh" remote="github" />
-  <project path="external/p7zip" name="CyanogenMod/android_external_p7zip" remote="github" />
-  <project path="external/pigz" name="CyanogenMod/android_external_pigz" remote="github" />
-  <project path="external/rsync" name="CyanogenMod/android_external_rsync" remote="github" />
-  <project path="external/stagefright-plugins" name="CyanogenMod/android_external_stagefright-plugins" remote="github" />
-  <project path="external/uicommon" name="CyanogenMod/android_external_cyanogen_UICommon" remote="github" />
-  <project path="external/unrar" name="CyanogenMod/android_external_unrar" remote="github" />
-  <project path="external/unzip" name="CyanogenMod/android_external_unzip" remote="github" />
-  <project path="external/vim" name="CyanogenMod/android_external_vim" remote="github" />
-  <project path="external/yaffs2" name="CyanogenMod/android_external_yaffs2" remote="github" />
-  <project path="external/zip" name="CyanogenMod/android_external_zip" remote="github" />
-  <project path="hardware/cyanogen" name="CyanogenMod/android_hardware_cyanogen" remote="github" />
-  <project path="frameworks/opt/hardware" name="CyanogenMod/android_frameworks_opt_hardware" remote="github" />
-  <project path="hardware/ti/omap4" name="CyanogenMod/android_hardware_ti_omap4" remote="github" />
-  <project path="hardware/ti/wlan" name="CyanogenMod/android_hardware_ti_wlan" remote="github" />
-  <project path="hardware/ti/wpan" name="CyanogenMod/android_hardware_ti_wpan" remote="github" />
-  <project path="packages/apps/AudioFX" name="CyanogenMod/android_packages_apps_AudioFX" remote="github" />
-  <project path="packages/apps/Browser" name="CyanogenMod/android_packages_apps_Browser" remote="github" />
-  <project path="packages/apps/CMBugReport" name="CyanogenMod/android_packages_apps_CMBugreport" remote="github" />
-  <project path="packages/apps/CMFileManager" name="CyanogenMod/android_packages_apps_CMFileManager" remote="github" />
-  <project path="packages/apps/CMParts" name="CyanogenMod/android_packages_apps_CMParts" remote="github" />
-  <project path="packages/apps/CMUpdater" name="CyanogenMod/android_packages_apps_CMUpdater" remote="github" />
-  <project path="packages/apps/CMWallpapers" name="CyanogenMod/android_packages_apps_CMWallpapers" remote="github" />
-  <project path="packages/apps/Eleven" name="CyanogenMod/android_packages_apps_Eleven" remote="github" />
-  <project path="packages/apps/Gello" name="CyanogenMod/android_packages_apps_Gello" remote="github" />
-  <project path="packages/apps/LockClock" name="CyanogenMod/android_packages_apps_LockClock" remote="github" />
-  <project path="packages/apps/Profiles" name="CyanogenMod/android_packages_apps_Profiles" remote="github" />
-  <project path="packages/apps/Screencast" name="CyanogenMod/android_packages_apps_Screencast" remote="github" />
-  <project path="packages/apps/SetupWizard" name="CyanogenMod/android_packages_apps_SetupWizard" remote="github" />
-  <project path="packages/apps/Snap" name="CyanogenMod/android_packages_apps_Snap" remote="github" />
-  <project path="packages/apps/ThemeChooser" name="CyanogenMod/android_packages_apps_ThemeChooser" remote="github" />
-  <project path="packages/apps/Trebuchet" name="CyanogenMod/android_packages_apps_Trebuchet" remote="github" />
-  <project path="packages/apps/WallpaperPicker" name="CyanogenMod/android_packages_apps_WallpaperPicker" remote="github" />
-  <project path="packages/providers/ThemesProvider" name="CyanogenMod/android_packages_providers_ThemesProvider" remote="github" />
-  <project path="packages/providers/WeatherProvider" name="CyanogenMod/android_packages_providers_WeatherProvider" remote="github" />
-  <project path="packages/services/CMAudioService" name="CyanogenMod/android_packages_services_CMAudioService" remote="github" />
-  <project path="packages/services/LiveLockScreenService" name="CyanogenMod/android_packages_services_LiveLockScreenService" remote="github" />
-  <project path="packages/services/ThemeManagerService" name="CyanogenMod/android_packages_services_ThemeManagerService" remote="github" />
-  <project path="packages/services/WeatherService" name="CyanogenMod/android_packages_services_WeatherService" remote="github" />
-  <project path="packages/themes/HexoLibre" name="CyanogenMod/android_packages_themes_HexoLibre" groups="themes" remote="github" />
-  <project path="prebuilts/cmsdk" name="CyanogenMod/android_prebuilts_cmsdk" clone-depth="1" revision="master" remote="github" />
-  <project path="system/extras/su" name="CyanogenMod/android_system_extras_su" remote="github" />
-  <project path="vendor/cm" name="CyanogenMod/android_vendor_cm" remote="github" />
-  <project path="vendor/cmsdk" name="CyanogenMod/cm_platform_sdk" remote="github" />
-  <project path="vendor/cmsdk/samples/weatherproviderservice/OpenWeatherMapProvider" name="CyanogenMod/android_packages_apps_OpenWeatherMapProvider" remote="github" />
-  <project path="vendor/cmsdk/samples/weatherproviderservice/WundergroundWeatherProvider" name="CyanogenMod/android_packages_apps_WundergroundWeatherProvider" remote="github" />
-  <project path="vendor/cmsdk/samples/weatherproviderservice/YahooWeatherProvider" name="CyanogenMod/android_packages_apps_YahooWeatherProvider" remote="github" />
+  <project path="android" name="CyanogenMod/android" />
+  <project path="external/ahbottomnavigation" name="CyanogenMod/android_external_ahbottomnavigation" />
+  <project path="external/android-visualizer" name="CyanogenMod/android_external_android-visualizer" />
+  <project path="external/bash" name="CyanogenMod/android_external_bash" />
+  <project path="external/cmsdk-api-coverage" name="CyanogenMod/android_external_cmsdk-api-coverage" />
+  <project path="external/exfat" name="CyanogenMod/android_external_exfat" />
+  <project path="external/ffmpeg" name="CyanogenMod/android_external_ffmpeg" />
+  <project path="external/fuse" name="CyanogenMod/android_external_fuse" />
+  <project path="external/gello-build" name="CyanogenMod/android_external_gello_build" />
+  <project path="external/google" name="CyanogenMod/android_external_google" />
+  <project path="external/htop" name="CyanogenMod/android_external_htop" />
+  <project path="external/koush/AndroidAsync" name="CyanogenMod/AndroidAsync" />
+  <project path="external/koush/ion" name="CyanogenMod/ion" />
+  <project path="external/libncurses" name="CyanogenMod/android_external_libncurses" />
+  <project path="external/libphonenumbergoogle" name="CyanogenMod/android_external_libphonenumbergoogle" />
+  <project path="external/libtar" name="CyanogenMod/android_external_libtar" />
+  <project path="external/libtruezip" name="CyanogenMod/android_external_libtruezip" />
+  <project path="external/lzo" name="CyanogenMod/android_external_lzo" />
+  <project path="external/nano" name="CyanogenMod/android_external_nano" />
+  <project path="external/ntfs-3g" name="CyanogenMod/android_external_ntfs-3g" />
+  <project path="external/openssh" name="CyanogenMod/android_external_openssh" />
+  <project path="external/p7zip" name="CyanogenMod/android_external_p7zip" />
+  <project path="external/pigz" name="CyanogenMod/android_external_pigz" />
+  <project path="external/rsync" name="CyanogenMod/android_external_rsync" />
+  <project path="external/stagefright-plugins" name="CyanogenMod/android_external_stagefright-plugins" />
+  <project path="external/uicommon" name="CyanogenMod/android_external_cyanogen_UICommon" />
+  <project path="external/unrar" name="CyanogenMod/android_external_unrar" />
+  <project path="external/unzip" name="CyanogenMod/android_external_unzip" />
+  <project path="external/vim" name="CyanogenMod/android_external_vim" />
+  <project path="external/yaffs2" name="CyanogenMod/android_external_yaffs2" />
+  <project path="external/zip" name="CyanogenMod/android_external_zip" />
+  <project path="hardware/cyanogen" name="CyanogenMod/android_hardware_cyanogen" />
+  <project path="frameworks/opt/hardware" name="CyanogenMod/android_frameworks_opt_hardware" />
+  <project path="hardware/ti/omap4" name="CyanogenMod/android_hardware_ti_omap4" />
+  <project path="hardware/ti/wlan" name="CyanogenMod/android_hardware_ti_wlan" />
+  <project path="hardware/ti/wpan" name="CyanogenMod/android_hardware_ti_wpan" />
+  <project path="packages/apps/AudioFX" name="CyanogenMod/android_packages_apps_AudioFX" />
+  <project path="packages/apps/Browser" name="CyanogenMod/android_packages_apps_Browser" />
+  <project path="packages/apps/CMBugReport" name="CyanogenMod/android_packages_apps_CMBugreport" />
+  <project path="packages/apps/CMFileManager" name="CyanogenMod/android_packages_apps_CMFileManager" />
+  <project path="packages/apps/CMParts" name="CyanogenMod/android_packages_apps_CMParts" />
+  <project path="packages/apps/CMUpdater" name="CyanogenMod/android_packages_apps_CMUpdater" />
+  <project path="packages/apps/CMWallpapers" name="CyanogenMod/android_packages_apps_CMWallpapers" />
+  <project path="packages/apps/Eleven" name="CyanogenMod/android_packages_apps_Eleven" />
+  <project path="packages/apps/Gello" name="CyanogenMod/android_packages_apps_Gello" />
+  <project path="packages/apps/LockClock" name="CyanogenMod/android_packages_apps_LockClock" />
+  <project path="packages/apps/Profiles" name="CyanogenMod/android_packages_apps_Profiles" />
+  <project path="packages/apps/Screencast" name="CyanogenMod/android_packages_apps_Screencast" />
+  <project path="packages/apps/SetupWizard" name="CyanogenMod/android_packages_apps_SetupWizard" />
+  <project path="packages/apps/Snap" name="CyanogenMod/android_packages_apps_Snap" />
+  <project path="packages/apps/ThemeChooser" name="CyanogenMod/android_packages_apps_ThemeChooser" />
+  <project path="packages/apps/Trebuchet" name="CyanogenMod/android_packages_apps_Trebuchet" />
+  <project path="packages/apps/WallpaperPicker" name="CyanogenMod/android_packages_apps_WallpaperPicker" />
+  <project path="packages/providers/ThemesProvider" name="CyanogenMod/android_packages_providers_ThemesProvider" />
+  <project path="packages/providers/WeatherProvider" name="CyanogenMod/android_packages_providers_WeatherProvider" />
+  <project path="packages/services/CMAudioService" name="CyanogenMod/android_packages_services_CMAudioService" />
+  <project path="packages/services/LiveLockScreenService" name="CyanogenMod/android_packages_services_LiveLockScreenService" />
+  <project path="packages/services/ThemeManagerService" name="CyanogenMod/android_packages_services_ThemeManagerService" />
+  <project path="packages/services/WeatherService" name="CyanogenMod/android_packages_services_WeatherService" />
+  <project path="packages/themes/HexoLibre" name="CyanogenMod/android_packages_themes_HexoLibre" groups="themes" />
+  <project path="prebuilts/cmsdk" name="CyanogenMod/android_prebuilts_cmsdk" clone-depth="1" revision="master" />
+  <project path="system/extras/su" name="CyanogenMod/android_system_extras_su" />
+  <project path="vendor/cm" name="CyanogenMod/android_vendor_cm" />
+  <project path="vendor/cmsdk" name="CyanogenMod/cm_platform_sdk" />
+  <project path="vendor/cmsdk/samples/weatherproviderservice/OpenWeatherMapProvider" name="CyanogenMod/android_packages_apps_OpenWeatherMapProvider" />
+  <project path="vendor/cmsdk/samples/weatherproviderservice/WundergroundWeatherProvider" name="CyanogenMod/android_packages_apps_WundergroundWeatherProvider" />
+  <project path="vendor/cmsdk/samples/weatherproviderservice/YahooWeatherProvider" name="CyanogenMod/android_packages_apps_YahooWeatherProvider" />
 
   <!-- CodeAurora additions -->
-  <project path="device/qcom/sepolicy" name="CyanogenMod/android_device_qcom_sepolicy" remote="github" />
-  <project path="external/ant-wireless/ant_native" name="CyanogenMod/android_external_ant-wireless_ant_native" remote="github" />
-  <project path="external/ant-wireless/ant_service" name="CyanogenMod/android_external_ant-wireless_ant_service" remote="github" />
-  <project path="external/ant-wireless/antradio-library" name="CyanogenMod/android_external_ant-wireless_antradio-library" remote="github" />
-  <project path="external/brctl" name="CyanogenMod/android_external_brctl" remote="github" />
-  <project path="external/connectivity" name="CyanogenMod/android_external_connectivity" remote="github" />
-  <project path="external/ebtables" name="CyanogenMod/android_external_ebtables" remote="github" />
-  <project path="external/gson" name="CyanogenMod/android_external_gson" remote="github" />
-  <project path="external/json-c" name="CyanogenMod/android_external_json-c" remote="github" />
-  <project path="external/libnetfilter_conntrack" name="CyanogenMod/android_external_libnetfilter_conntrack" remote="github" />
-  <project path="external/libnfnetlink" name="CyanogenMod/android_external_libnfnetlink" remote="github" />
-  <project path="external/mm-dash" name="CyanogenMod/android_external_mm-dash" remote="github" />
-  <project path="external/powertop" name="CyanogenMod/android_external_powertop" remote="github" />
-  <project path="external/protobuf-c" name="CyanogenMod/android_external_protobuf-c" remote="github" />
-  <project path="system/qcom" name="CyanogenMod/android_system_qcom" groups="qcom" remote="github" />
-  <project path="vendor/qcom/opensource/bluetooth" name="CyanogenMod/android_packages_apps_BluetoothExt" remote="github" />
+  <project path="device/qcom/sepolicy" name="CyanogenMod/android_device_qcom_sepolicy" />
+  <project path="external/ant-wireless/ant_native" name="CyanogenMod/android_external_ant-wireless_ant_native" />
+  <project path="external/ant-wireless/ant_service" name="CyanogenMod/android_external_ant-wireless_ant_service" />
+  <project path="external/ant-wireless/antradio-library" name="CyanogenMod/android_external_ant-wireless_antradio-library" />
+  <project path="external/brctl" name="CyanogenMod/android_external_brctl" />
+  <project path="external/connectivity" name="CyanogenMod/android_external_connectivity" />
+  <project path="external/ebtables" name="CyanogenMod/android_external_ebtables" />
+  <project path="external/gson" name="CyanogenMod/android_external_gson" />
+  <project path="external/json-c" name="CyanogenMod/android_external_json-c" />
+  <project path="external/libnetfilter_conntrack" name="CyanogenMod/android_external_libnetfilter_conntrack" />
+  <project path="external/libnfnetlink" name="CyanogenMod/android_external_libnfnetlink" />
+  <project path="external/mm-dash" name="CyanogenMod/android_external_mm-dash" />
+  <project path="external/powertop" name="CyanogenMod/android_external_powertop" />
+  <project path="external/protobuf-c" name="CyanogenMod/android_external_protobuf-c" />
+  <project path="system/qcom" name="CyanogenMod/android_system_qcom" groups="qcom" />
+  <project path="vendor/codeaurora/telephony" name="CyanogenMod/android_vendor_codeaurora_telephony" />
+  <project path="vendor/qcom/opensource/bluetooth" name="CyanogenMod/android_packages_apps_BluetoothExt" />
 </manifest>
-- 
2.17.1

